src/mainboard/starlabs/labtop/variants/cml/devicetree.cb

```diff
@@ -90,29 +75,44 @@ chip soc/intel/cannonlake
 		device pci 04.0 on				# SA Thermal Device
 			register "Device4Enable" = "1"
 		end
-		device pci 08.0 on  end				# Gaussian Mixture Model
 		device pci 12.0 on  end				# Thermal Subsystem
 		device pci 12.5 off end				# UFS SCS
 		device pci 12.6 off end				# GSPI #2
 		device pci 14.0 on				# USB xHCI
 			# USB2
-			register "usb2_ports[0]" = "USB2_PORT_TYPE_C(OC2)"	# Type-C Port 1
+			register "usb2_ports[0]" = "USB2_PORT_MID(OC2)"		# Type-C Port 1
 			register "usb2_ports[1]" = "USB2_PORT_MID(OC_SKIP)"	# Type-A Port 2
 			register "usb2_ports[2]" = "USB2_PORT_MID(OC2)"		# Bluetooth
 			register "usb2_ports[3]" = "USB2_PORT_MID(OC2)"		# SD Card
+			register "usb2_ports[4]" = "USB2_PORT_MID(OC3)"
 			register "usb2_ports[5]" = "USB2_PORT_MID(OC3)"		# Type-A Port 3
 			register "usb2_ports[6]" = "USB2_PORT_MID(OC3)"		# Camera
+			register "usb2_ports[7]" = "USB2_PORT_MID(OC3)"
+			register "usb2_ports[8]" = "USB2_PORT_MID(OC_SKIP)"
 			register "usb2_ports[9]" = "USB2_PORT_MID(OC_SKIP)"	# CNVi Bluetooth
-
+			register "usb2_ports[10]" = "USB2_PORT_MID(OC_SKIP)"
```

Comment:
Aren't these enabling the unused USB host ports unnecessarily?

Reply:
Yup - they're the AMI settings.